doc/new-book/configuration-security.adoc

```diff
@@ -373,22 +373,26 @@ This is the service principal that {RouterName} uses.
 .Procedure
 
 . On the router's host machine, open the `/etc/sasl2/qdrouterd.conf` configuration file.
-
-. Verify that the `mech_list` attribute contains the `GSSAPI` mechanism.
 +
 --
-.The `/etc/sasl2/qdrouterd.conf` Configuration File
+.An `/etc/sasl2/qdrouterd.conf` Configuration File
 ====
 [options="nowrap"]
 ----
 pwcheck_method: auxprop
 auxprop_plugin: sasldb
 sasldb_path: qdrouterd.sasldb
+keytab: /tmp/keytabs/server.keytab
```

Comment:
Probably not `/tmp/`, though. That would be silly thing to do in actual deployment for the fact that RHELs auto-delete old files under `/tmp`. There are AFAIK supposed to be permissions related security issues around using /tmp, but I am now unable to find more on this.

Reply:
:+1: true @jdanekrh